app/main.py

```diff
@@ -1 +1,70 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name):
+        self.name = name
```

Comment:
put skills into init, so when you inherit for FullStackDeveloper you will not duplicate code